src/runtime/config/configuration-qemu-snp.toml.in

```diff
@@ -705,4 +709,4 @@ image_request_timeout = @DEFIMAGEREQUESTTIMEOUT@
 #
 # Keys can be remotely provisioned. The Kata agent fetches them from e.g.
 # a HTTPS URL:
-#provision=https://my-key-broker.foo/tenant/<tenant-id>
\ No newline at end of file
+#provision=https://my-key-broker.foo/tenant/<tenant-id>
```

Comment:
btw, maybe it's time for us to get rid of this placeholder section? seems like it could be very confusing as we start to implement more features 

wdyt @arronwy @stevenhorsman ?

Reply:
Yeah, that image section pre-dates my involvement in the project, so is in `main`, not just `CCv0`, but should definitely be reviewed as part of the image pull -> main contribution. We definitely use the `service_offload` value and I think we also use `image_request_timeout`, but I don't think the provision has ever been used, or makes much sense, so I'd expect it to go

Reply:
yes, fully agree, it is confusing to new user since runtime stack is not using them.